### PR TITLE
Maxbader indigo devel tricycledrive enchanted

### PR DESCRIPTION
**Pull request by hsu**
attempt at fixing conflicts of #292

### CODE REVIEW

File: gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp [hunk 1]
```diff
@@ -227,17 +227,15 @@ void GazeboRosTricycleDrive::motorController ( double target_speed, double targe
...
         if ( fabs ( diff_speed ) < wheel_speed_tolerance_ ) {
-            applied_speed = target_speed;
-        } else if ( diff_speed < target_speed ) {
-            applied_speed = current_speed + wheel_acceleration_ * dt;
-        } else {
-            applied_speed = current_speed - wheel_deceleration_ * dt;
+            applied_speed = current_speed;
+        } else if ( fabs(diff_speed) > wheel_acceleration_ * dt ) {
+            if(diff_speed > 0){ applied_speed = current_speed - wheel_acceleration_ * dt;}
+            else              { applied_speed = current_speed + wheel_deceleration_ * dt;}
```
Comment (scpeters) [inline gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp]: did you want to fix the indentation you commented about in #292 ?

**Comment by hsu** [comment]
I don't quite like using SetVelocity because of unrealistic impulses, or SetPosition because it skips dynamics. But it's an improvement of the state of the plugin.

Ideally, this should be rewritten using SetForce to control steering / drive if the need arises.

File: gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp (hunk 1 above)
Comment (hsu) [inline gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp]: I fixed the indents, and I reverted to Gazebo style in some places due to lack of self restraint :)
but given things are not formatted uniformly in gazebo_ros_pkgs right now, we should probably deal with stylistic issues in a separate global effort.

Comment (scpeters) [inline gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp]: I was just reminding you of a comment you made. I don't think we should make huge style changes until we declare what the style should be for these packages

**Comment by scpeters** [comment]
+1

**Pull request merged**